lstchain/io/io.py

```diff
@@ -268,22 +267,29 @@ def merging_check(file_list):
     assert len(file_list) > 1, "The list of files is too short"
     mergeable_list = file_list.copy()
 
-    filename0 = mergeable_list[0]
-    array_info0 = read_array_info(filename0)
-    mcheader0 = read_simu_info_hdf5(filename0)
-    thrown_events_hist0 = read_simtel_energy_histogram(filename0)
-    metadata0 = read_metadata(filename0)
+    first_file = mergeable_list[0]
+    subarray_info0 = SubarrayDescription.from_hdf(first_file)
+    metadata0 = read_metadata(first_file)
+
+    if subarray_info0.name == "MonteCarloArray":
```

Comment:
Is there a way of knowing if a file is simulated or not better than this?

Reply:
good point
there is a metadata `is_simulation` in event_source
This could be added to the `global_metadata`.

Reply:
i think this way of checking is fine for now, until we adopt ctapipe DL1